petastorm/unischema.py

```diff
@@ -401,19 +401,27 @@ def insert_explicit_nulls(unischema, row_dict):
                 raise ValueError('Field {} is not found in the row_dict, but is not nullable.'.format(field_name))
 
 
+def _fullmatch(regex, string, flags=0):
```

Comment:
Since Petastorm is supported for both Python 2 and 3, won't this wind up using doing the emulated version even when `re.fullmatch()` is supported? My worry here is that there might be weird edgecases that `fullmatch()` handles (regular expressions are complicated, so I think this is a reasonable concern) and our implementation might not. Could we change the implementation here to something like:
```
def _fullmatch(regex, string, flags=0):
  if `'fullmatch' in dir(re):
    re.fullmatch(regex, string, flags)
  else:
    # existing code below 
```

Reply:
Done